app/models/katello/host_available_module_stream.rb

```diff
@@ -48,6 +48,19 @@ def install_status
       end
     end
 
+    def self.installed_status(status, host)
+      case status
+      when 'not installed'
+        where(installed_profiles: [])
+      when 'installed'
```

Comment:
I am getting this logic from here, not sure if it's accurate..will need some eyes on this:
```
switch (true) {
  case (streamInstallStatus?.length > 0 && streamText === 'disabled'):
    return <TableText wrapModifier="nowrap">{INSTALLED_STATE.INSTALLED}</TableText>;
  case (streamInstallStatus?.length > 0 && streamText === 'enabled' && upgradable !== true):
    return <><CheckIcon color="green" /> {INSTALLED_STATE.UPTODATE}</>;
  case (streamInstallStatus?.length > 0 && streamText === 'enabled' && upgradable):
    return <><LongArrowAltUpIcon color="blue" /> {INSTALLED_STATE.UPGRADEABLE}</>;
  default:
    return <InactiveText text={INSTALLED_STATE.NOTINSTALLED} />;
  }
};
```

Reply:
@chris1984 do you remember where this logic came from? Looks like it was introduced here: https://github.com/Katello/katello/pull/9911

Reply:
@ianballou @sjha4 

Yeah it was, I worked with Partha to come up with the formula on it since the new field was added by Maria and to show that we had to compare a few fields in the table.

Reply:
Sounds good..I have tried to replicate the formula in the API instead to be able to filter directly on it. Will remove installed from the Installed status filter..That makes sense to me.